lib/action_controller/serialization/context.rb

```diff
@@ -0,0 +1,12 @@
+module ActionController
```

Comment:
I agree - action controller isn't ever going to really make use of the context, where as it is AMS that utilizes context.